openmc/tallies.py

```diff
@@ -587,8 +672,20 @@ def get_tally_xml(self):
 
         return element
 
-
     def find_filter(self, filter_type):
+        """Return a filter in the tally that matches a specified type
```

Comment:
ditto my earlier comment - should any mention be made that ``None`` will be returned if no matching ``Filter`` is found?